src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -486,10 +486,5 @@ private string GetClientSpecifiedSpn()
             }
             return spn;
         }
-
-        private int DecryptNtlm(byte[] payload, int offset, int count, out int newOffset, uint expectedSeqNumber)
```

Comment:
Yes.. Negotiate protocol works .I will change the function in .windows to private.